apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -212,24 +252,424 @@ describe('MultiPayment', () => {
+                            })
+                        }, MULTIPAYMENT_FROZEN_FIELD_INCLUDED)
+                    })
+                })
+                describe('Value-fields', () => {
+                    const cases = statuses.map(status => {
+                        return MULTIPAYMENT_FROZEN_FIELDS[status]
+                            .filter(field => values.includes(field))
+                            .map(field => [status, field])
+                    }).flat(1)
+                    test.each(cases)('Old status: %p, Field: %p', async (status, field) => {
+                        const { admin, client, payments, acquiringIntegration } = await makePayerAndPayments()
+                        const [multiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration, {
+                            status,
+                        })
+                        await expectToThrowValidationFailureError(async () => {
+                            await updateTestMultiPayment(admin, multiPayment.id, {
+                                status: MULTIPAYMENT_ERROR_STATUS,
+                                [field]: get(multiPayment, field, null),
+                            })
+                        }, MULTIPAYMENT_FROZEN_FIELD_INCLUDED)
+                    })
+                })
+            })
+            describe('Cannot be created if any of payments', () => {
+                test('Has not init status', async () => {
+                    const { admin, billingReceipts, organization, acquiringContext, client, acquiringIntegration } = await makePayer(1)
+                    const [payment] = await createTestPayment(admin, organization, billingReceipts[0], acquiringContext, {
+                        status: PAYMENT_DONE_STATUS,
+                    })
+                    await expectToThrowValidationFailureError(async () => {
+                        await createTestMultiPayment(admin, [payment], client.user, acquiringIntegration)
+                    }, MULTIPAYMENT_NON_INIT_PAYMENTS)
+                })
+                test('Already has multipayment', async () => {
+                    const { admin, client, acquiringIntegration, payments } = await makePayerAndPayments()
+                    await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
+                    await expectToThrowValidationFailureError(async () => {
+                        await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
+                    }, MULTIPAYMENT_PAYMENTS_ALREADY_WITH_MP)
                 })
-            }, 'unknownDataVersion')
-            const [multiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
-            await expectToThrowValidationFailureError(async () => {
-                await updateTestMultiPayment(admin, multiPayment.id, {
-                    dv: 2,
+            })
+            // NOTE: APPROVED BY ALEXANDER
```

Comment:
I'd rather left a description of what was approved and why -- I think nobody, but you and Alexander understand this comment ))

Reply:
It means, that all logic  cases in it's describe block was approved by Alexander, but I added more clear description